mesonbuild/mesondata.py

```diff
@@ -24,13 +25,16 @@ def write_once(self, path: Path) -> None:
             path.write_text(data, encoding='utf-8')
 
     def write_to_private(self, env: 'Environment') -> Path:
-        try:
-            resource = importlib.resources.files('mesonbuild') / self.path
-            if isinstance(resource, Path):
-                return resource
-        except AttributeError:
-            # fall through to python 3.7 compatible code
-            pass
+        if sys.version_info >= (3, 10):
```

Comment:
It should be fine with 3.9 too?

Reply:
The error in the previous lint run was occurring for 3.7 and 3.8 only -- and it said "files() doesn't exist".